webui/react/config-overrides.js

```diff
@@ -51,7 +49,7 @@ const webpackConfig = override(
         // TODO: Near future, add more colors to override in browser dynamically.
       ],
       indexFileName: 'index.html',
-      publicPath: PUBLIC_URL,
+      publicPath: process.env.PUBLIC_URL,
```

Comment:
**question:** does this pick up the envvar `PUBLIC_URL` without passing in `PUBLIC_URL=X` via `npm (start|run build)`?